lib/rest_in_peace.rb

```diff
@@ -47,10 +47,8 @@ def clear_changes
   end
 
   def update_attributes(attributes)
-    attributes.each do |key, value|
-      next unless respond_to?("#{key}=")
-      send("#{key}=", value)
-    end
+    warn 'DEPRECATION WARNING: the method update_attributes is deprecated, use method update.'
```

Comment:
https://redmine.nine.ch/issues/32737#Acceptance-Criteria last point, David was required to have `deprecated message`